src/main/java/filter/expression/Qualifier.java

```diff
@@ -97,6 +97,16 @@ public static FilterExpression replaceMilestoneAliases(IModel model, FilterExpre
                 .sorted((a, b) -> getMilestoneDueDateComparator().compare(a, b))
                 .collect(Collectors.toList());
 
+        List<TurboMilestone> openMilestones = model.getMilestones().stream()
+                .filter(ms -> ms.isOpen())
+                .collect(Collectors.toList());
+
+        if (openMilestones.size() == 1 && !openMilestones.get(0).getDueDate().isPresent()) {
+            //if there is only one open issue in the repo and it doesn't have a due date, add it
+            //to the list of all milestones
+            allMilestones.addAll(openMilestones);
+        }
```

Comment:
I think we need it, because the case where a milestone without due date can be considered by the alias `current` is when there is no open milestone with due date, right? Is it the intended behaviour?

For the second part, I think we can just create another method to generate the `allMilestones` list.